backend/app/validators/array_inclusion_validator.rb

```diff
@@ -4,6 +4,8 @@ class ArrayInclusionValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.nil?
 
+    value.reject!(&:blank?)
```

Comment:
Probably this is not a good place to do this, but didn't want to pollute controllers with eliminating all of those blank values that we get by default, in case someone wants to unselect all categories for example. Rails is sending empty value in array unless include_blank is set to `false`. It is sending for example ["", "cat1", "cat2"], just in case if someone unselect cat1, cat2 to send this parameter, otherwise categories will be unchanged.

Reply:
Yeah, if put here, it is gonna be a bit of magic at background, a bit difficult to find for our future selfs :).

Reply:
I think it can stay here as this is a custom validator, we can treat it as a kind of  automatic `before_validation` hook, as this is one of the places where that kind of logic usually exists.